src/templates/react/Content/src/App.fs

```diff
@@ -0,0 +1,149 @@
+namespace FableElmishReactTemplate
+
+open Elmish
+open Elmish.Browser.Navigation
+open Elmish.Browser.UrlParser
+open Fable.Core
+open Fable.Core.JsInterop
+open Fable.Import
+open Fable.Import.Browser
+open FableElmishReactTemplate.Common
+
+module App =
+
+  importAll "../sass/main.sass"
+
+  type SubModels =
+    {
+      Home: Home.Model
```

Comment:
Since interop with .NET is not an issue (the source of CamelCase notation in F#), I think we should use JS (lowerCase) conventions when it comes to names.